app/src/main/java/com/gophillygo/app/data/models/Filter.java

```diff
@@ -0,0 +1,149 @@
+package com.gophillygo.app.data.models;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Filter implements Parcelable {
+    public boolean nature, exercise, educational, been, wantToGo, notInterested, liked, accessible;
+
+    public Filter() {
+        this(false, false, false, false, false, false, false, false);
+    }
+
+    public Filter(boolean nature, boolean exercise, boolean educational, boolean been,
+                  boolean wantToGo, boolean notInterested, boolean liked, boolean accessible) {
+        this.nature = nature;
+        this.exercise = exercise;
+        this.educational = educational;
+        this.been = been;
+        this.wantToGo = wantToGo;
+        this.notInterested = notInterested;
+        this.liked = liked;
+        this.accessible = accessible;
+    }
+
+    private Filter(Parcel in) {
+        boolean[] fields = in.createBooleanArray();
+        this.nature = fields[0];
```

Comment:
Strongly recommend a refactor for this field mapping; these integer field offsets look pretty brittle.

Would be great to not have to rely on field order to pass all eight of these flags. Or at least, define some static mapping of flag to position.